mongo/client_examples_test.go

```diff
@@ -314,3 +314,50 @@ func ExampleConnect_aWS() {
 	}
 	_ = ecClient
 }
+
+func ExampleConnect_versionedAPI() {
+	// Configure a Client with versioned API.
+	//
+	// Versioned API is a new feature in MongoDB 5.0 that allows user-selectable API versions, subsets of MongoDB
+	// server semantics, to be declared on a Client. During communication with a server, Clients with a declared
+	// API version will force that server to behave in a manner compatible with the API version. Declaring an API
+	// version on your Client can be used to ensure consistent responses from a server, providing long term API
+	// stability for an application.
+	//
+	// The declared API version is applied to all commands run through the Client, including those sent through
+	// the generic RunCommand helper. Specifying versioned API options in the command document AND declaring an API
+	// version on the Client is not supported and will lead to undefined behaviour. To run any command with a different
```

Comment:
tiniest nit: behaviour -> behavior
probably want to do American spellings?

Reply:
Agree; much to Andreas' chagrin.